website/addons/dryad/static/node-cfg.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var $ = require('jquery');
+require('./dryad-node-config.js');
+var bootbox = require('bootbox');
+var AddonHelper = require('js/addonHelper');
+
+
+    $(document).ready(function() {
```

Comment:
Please unindent this block and try to make sure indention level is consistent throughout this file.